samples/petstore/microservices/fileserver/src/main/java/org/wso2/msf4j/examples/petstore/fileserver/Runner.java

```diff
@@ -25,11 +25,15 @@
  */
 public class Runner {
 
+    private Runner() {
```

Comment:
Please see the [above](https://github.com/wso2/msf4j/pull/338#discussion_r105387166) comment. Thanks!